src/webview/handleOutboundEvents.js

```diff
@@ -267,9 +268,13 @@ export const handleWebViewOutboundEvent = (
       handleImage(props, event.src, event.messageId);
       break;
 
-    case 'longPress':
+    case 'longPress': {
+      if (Platform.OS === 'android') {
```

Comment:
Why is this Android only? We should have haptic feedback on iOS as well.

Reply:
> Inside the WebView, on iOS. This one seems to already work (see chat discussion) on iOS 13+. No action needed, 

said by @gnprice  [here](https://github.com/zulip/zulip-mobile/issues/4649#issue-856264579) in comment :)

Reply:
Ah, I see. Please add a comment stating that. Particularly, we should note that this means that haptic feedback will not work on iOS <13.

I would add a `longPressHapticFeedback` function in this file, so that you can have the check and the comment only in one place. This will also be useful if we, for instance, choose to [detect the iOS version](https://reactnative.dev/docs/platform-specific-code#detecting-the-ios-version).